libs/shared/components/src/auth/UserMenu/UserMenu.css.ts

```diff
@@ -19,6 +19,25 @@ export const dropdown = style({
   }),
 })
 
+export const fullScreen = style({
+  position: 'fixed',
```

Comment:
This causes a regression in application system form, where the user menu button is part of the grid but the modal appears in the corner.

Before:
![image](https://user-images.githubusercontent.com/115094/146946344-97d10e1d-1b95-4e65-bac1-d1aea8d88c43.png)

After:
![image](https://user-images.githubusercontent.com/115094/146946159-f3f96824-f911-4e5b-a56f-4f9de5fc9a5a.png)

Reply:
Ohno - I'll fix

Reply:
Done ✅